src/common/home/Sponsors.jsx

```diff
@@ -0,0 +1,157 @@
+              className="partners-sponsors-cta"
+              href="https://forms.gle/19KTxhzp4aiTJ4kMA"
+              rel="noreferrer"
+              target="_blank"
+            >
+              Partner an Event <FaHandshake className="handshake" size={28} />
+            </a>
+          </div>
+        </div>
+        <div className="event-partners-sponsors">
+          <h3>Event Sponsors</h3>
+          <div className="content">
+            <div className="partners-sponsors-grid">
+              {eventSponsors.map((s) => (
+                <a
+                  className="partners-sponsors-grid-item"
+                  href={s.website}
+                  rel="noreferrer"
+                  target="_blank"
+                >
+                  <img alt={s.name} src={s.logo} />
+                </a>
+              ))}
+            </div>
+            <a
+              className="partners-sponsors-cta"
+              href="https://forms.gle/19KTxhzp4aiTJ4kMA"
+              rel="noreferrer"
+              target="_blank"
+            >
+              Sponsor an Event <FaHandshake className="handshake" size={28} />
+            </a>
+          </div>
+        </div>
+      </div>
+      {/* NOTE: In the following two sections, there are inline styles to override the custom css classes. These are being used for the time being, until data for the sections are available. */}
+      <div className="backers-container">
+        <h3>Backers</h3>
+        <span>
+          Support us with any monthly sponsorship. Your help means a lot to us.
+          <br /> We will give you a shoutout and add you here.
+        </span>
+        <div
+          className="backers-grid"
+          style={{
+            display: 'flex',
+            flexDirection: 'column',
+            justifyContent: 'start',
+            alignItems: 'center'
+          }}
+        >
+          {/* {backers.map((b) => (
+            <a className="backers-grid-item" rel="noreferrer" target="_blank">
+              abcd
+            </a>
+          ))} */}
+          <a
+            className="backers-cta"
+            href="https://github.com/sponsors/reactplay"
+            rel="noreferrer"
+            target="_blank"
+          >
+            <RiHeartFill className="heart" size={28} />
+            <RiHeartAddFill className="heart-add" size={28} />
+          </a>
+        </div>
+      </div>
+      <div className="prime-sponsors-container">
+        <h3>Prime Sponsors</h3>
+        <span>
+          Be a prime sponsor of ReactPlay. <br />
+          Your sponsorship will go a long way, and we will place you in our internal events,
+          newsletters and here on the homepage.
+        </span>
+        <div
+          className="prime-sponsors-grid"
+          style={{
+            display: 'flex',
+            flexDirection: 'column',
+            justifyContent: 'start',
+            alignItems: 'center'
+          }}
+        >
+          {/* {primeSponsors.map((p) => (
```

Comment:
Remove it if not required

Reply:
When we start getting sponsors, we will need this.